Tasks/Loktionova/Task2/task2.py

```diff
@@ -0,0 +1,46 @@
+import datetime
+
+mode = input ('Do you want to know the current time? Print Yes if yes?: ')
+if mode == 'Yes':
+    time = datetime.datetime.now().strftime('%H:%M').split(':')
+else:
+    time = input ('Enter your time in format HH:MM: ').split(':')
+hour = int(time[0])
+min = int(time[1])
+
+num_min = {0:('минут', 'минуты'), 1:('одна минута', 'одной минуты'), 2:('две минуты','двух минут'), 3:('три минуты','трех минут'), 
+4:('четыре минуты','четырех минут'), 5:('пять минут','пяти минут'), 6:('шесть минут','шести минут'), 7:('семь минут', 'семи минут'), 
+8:('восемь минут','восьми минут'), 9:('девять минут','девяти минут'), 10:('десять минут','десяти минут'),
+11:('одиннадцать минут','одиннадцати минут'), 12:('двенадцать минут','двенадцати минут'), 13:('тринадцать минут', 'тринадцати минут'), 
+14:('четырнадцать минут','четырнадцати минут'), 15:('пятнадцать минут','пятнадцати минут'),
+16:('шестнадцать минут', 'шестнадцати'), 17:('семнадцать минут', 'семнадцати'), 18:('восемнадцать минут', 'восемнадцати'),
+19:('девятнацать минут', 'девятнадцати'), 20:('двадцать', 'двадцати'), 30:('тридцать', 'тридцати'), 40:('сорок', 'сорока')}
+num_hour = {0:('ноль часов', 'первого') , 1:('один час', 'первого'), 2:('два часа','второго'), 3:('три часа','третьего'), 
+4:('четыре часа','четвертого'), 5:('пять часов','пятого'), 6:('шесть часов','шестого'), 7:('семь часом', 'седьмого'), 
+8:('восемь часов','восьмого'), 9:('девять часов','девятого'), 10:('десять часов','десятого'), 11:('одиннадцать часов','одиннадцатого'), 
+12:('двенадцать часов','двенадцатого'), 13:('тринадцать часов','первого')}
+
+if 12 < hour < 24:
+    hour -= 12
+
+if hour > 24:
+    print ('Data is not correct')
+    quit()
```

Comment:
these two checks would work better in a different order, check for the **hour > 24**, and then for the **12 < hour < 24**, not a huge mistake, -1 point